main/ertp/api/payment.md

```diff
@@ -1,24 +1,67 @@
+new `purse`. The deposit operation destroys the `payment`.
 
 ## payment.getAllegedBrand()
 - Returns: `{Brand}`
 
-Get the allegedBrand, indicating the kind of digital asset this payment purports to be, and which issuer to use. Because payments are not trusted, any method calls on payments should be treated with suspicion and verified elsewhere.
+Get the allegedBrand, indicating the kind of digital asset this `payment` purports to be, and which `issuer` to use 
+with it. Because `payments` are not trusted, any method calls on `payments` 
+should be treated with suspicion and verified elsewhere.
 
 ```js
-const { issuer, mint, brand, amountMath } = makeIssuerKit('bucks');
-const payment = mint.mintPayment(amountMath.make(10));
-
+const { issuer: quatloosIssuer, mint: quatloosMint, 
+        brand: quatloosBrand, amountMath: quatloosAmountMath } = makeIssuerKit('quatloos');
+const payment = quatloosMint.mintPayment(quatloosAmountMath.make(10));
+//Should return 'quatloos'
 const officialBrand = payment.getAllegedBrand();
 ```
+
+## Related Methods
+
+The following methods on other ERTP components also either operate
+on or return a `brand`. While a brief description is given for each, 
+you should click through to a method's main documentation entry for 
+full details on what it does and how to use it.
+
+- [`issuer.getAmountOf(payment)`](./issuer.html#issuer-getamountof-payment)
+  - Get a description of a `payment` balance as an amount`. 
+- [`issuer.burn(payment, optAmount)`](./issuer.html#issuer-burn-payment-optamount)
+  - Burn (destroy) all of the digital assets in the `payment`.
+- [`issuer.claim(payment, optAmount)`](./issuer.html#issuer-claim-payment-optamount)
+  - Transfer all assets from the `payment` to a returned new `payment` and burn the original.
+- [`issuer.combine(paymentsArray)`](./issuer.html#issuer-combine-paymentsarray)
+  - Combine multiple `payments` into one, returned, `payment`.
+- [`issuer.split(payment, paymentAmountA)`](./issuer.html#issuer-split-payment-paymentamounta)
+  - Split one `payment` into two new ones.
+- [`issuer.splitMany(payment, amountArray)`](./issuer.html#issuer-splitmany-payment-amountarray)
+  - Split `payment` into multiple `payments`, returned as an array.
+- [`issuer.isLive(payment)`](./issuer.html#issuer-islive-payment)
+  - Returns `true` if `payment` has value. 
+- [`mint.mintPayment(newAmount)`](./mint.html#mint-mintpayment-newamount)
+  - Returns a new `payment` containing the newly minted assets corresponding to the `newAmount` argument. 
+- [`purse.deposit(payment, optAmount`)](./purse.html#purse-deposit-payment-optamount)
+  - Deposit all of `payment` into this `purse`.
+- [`purse.makeDepositFacet()`](./purse.html#purse-makedepositfacet)
+  - Creates a deposit-only facet on the `purse` that can be given to other parties.
+- [`purse.withdraw(amount)`](./purse.html#purse-withdraw-amount)
+  - Returns a new `payment` whose balance is described by the `amount` argument. 
```

Comment:
The link anchors don't seem to work. At least my browser (Brave or Chrome) doesn't scroll to the link on the destination page.  Also the links don't render as visible links because of the code formatting, until you hover.  I don't know how to fix this other than to do something like

```
- [burn](./issuer.html#issuer-burn-payment-optamount) `issuer.burn(payment, optAmount)`
  - Burn (destroy) all of the digital assets in the `payment`.
```

Maybe the description "you should click through to ..." is sufficient.